target.py

```diff
@@ -1,24 +1,40 @@
-import pygame
-import random
+from constants import *
+from enum import Enum
 
 
-class Target(pygame.sprite.Sprite):
+class TargetAttr(Enum):
```

Comment:
nice use of enums

Reply:
I didn't know enums had to be imported in python!

Reply:
Yeah, they're quite different than in C++ that I know of.